apps/aecore/src/aec_chain_state.erl

```diff
@@ -1158,3 +1165,17 @@ calc_rewards(FraudStatus1, FraudStatus2, GenerationFees,
     LockedAmount = TotalBlockAmount - B1Amt - B2Amt,
     {B1Amt, B2Amt, LockedAmount}.
 
+reward_split_for_protocol_foundation(BeneficiaryReward1, BeneficiaryReward2, NewestNodeHeight) ->
+    ActivationHeight = aec_governance:protocol_beneficiary_activation(NewestNodeHeight),
+    ActivationFlag = aec_governance:protocol_beneficiary_enabled(),
+    if ActivationHeight and ActivationFlag ->
+        ContribFactor = aec_governance:protocol_beneficiary_factor(),
+        Contrib1 = BeneficiaryReward1 * ContribFactor div 1000, %% todo: drop zeros, div 100 after merges
```

Comment:
How can this TODO be addressed? (Not clear to me what it is meant to track.)

Reply:
I will remove it.